components/Table/TableContainer.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import HardwareKeyboardArrowRight from 'material-ui/svg-icons/hardware/keyboard-arrow-right';
 import styles from './TableContainer.css';
 
 const TableContainer = ({ title, style, children }) => (
   <div className={styles.container} style={{ ...style }}>
     <div className={styles.heroesContainer}>
-      <div className={styles.tableHeading}>{title}</div>
+      <div className={styles.tableHeading}>
+        <HardwareKeyboardArrowRight style={{ verticalAlign: 'top', opacity: '.6', height: 26 }} />
+        {title}
+      </div>
```

Comment:
without icon it look not as I would like to :art:
changed to another

![image](https://cloud.githubusercontent.com/assets/20866892/18673702/7641179c-7f55-11e6-834e-2d0e9383fb2a.png)